assets/app/view/game/dividend.rb

```diff
@@ -52,8 +52,8 @@ def render
               'None'
             end
 
-          if entity.loans.any? && !@game.can_pay_interest?(entity, corp_income)
-            text += ' (Liquidate)'
+          if entity.loans.any? && !@game.can_pay_interest?(entity, corp_income) && @game.cannot_pay_interest_warning_str
```

Comment:
mabye just call it cannot_pay_interest_str

Reply:
I've changed the name